tools/beerocks_analyzer/doc/README.org

```diff
@@ -0,0 +1,71 @@
+#+OPTIONS: toc:nil \n:t
```

Comment:
I don't think it makes a lot of sense to include a .org in the repository, since a .md works just as well and that's what we standardised on. So I removed it. Feel free to put it back if you feel it adds value (but then defend why :-).